### ISSUE TITLE
[Enhancement]To long text on HUD

### ISSUE THREAD

**Issue by criss20ster** [Courseplay/Courseplay_FS22#705]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
The problem with the appearance of the text as you can see in the picture, the text overlaps the CP functions to change the tool +/- 

**Actual behavior**
The problem with the appearance of the text as you can see in the picture, the text overlaps the CP functions to change the tool +/- 

** Expected behavior **
Improving the visibility of the text 

**Screenshots**
(https://i.imgur.com/1WZ8ibL.jpeg)

**Courseplay version**
7.0.0.22

**Log**
https://pastebin.com/Swj2YmaM

**Map**
Elmcreek

**Vehicles/ implements involved**
RTx tractor + Kverneland plow 

mod Yes

**Relevant script mods**
Autodrive

**Comment by Tensuko** [comment]
This is a problem of the Translation.
Is there a way to shorten the text ?

**Comment by criss20ster** [comment]
Well, how can I describe these activities in a shorter amount to know what it is and how it serves

**Comment by Tensuko** [comment]
Offset left/right
Offset front/back 
That is up to you how it fits in your language.
Just discrube what it is for.

**Comment by criss20ster** [comment]
> Offset left/right
> Offset front/back
> That is up to you how it fits in your language.
> Just discrube what it is for.

And which configuration is responsible for the stretching of the screen, I would pull myself under myself

**Comment by Tensuko** [comment]
Scaling, that is an ingame setting of the basegame.

**Comment by criss20ster** [comment]
> Scaling, that is an ingame setting of the basegame.

ok But in the game there is no scaling option in the settings

**Comment by Tensuko** [comment]
![fsScreen_2022_01_26_12_10_43](https://user-images.githubusercontent.com/27344744/151153098-e503095e-9bb7-46a5-91d8-e40904e01a35.png)

Best way for your us to reduce the length of the Translation.
I mean there is sure a shorter way to discribe that.

**Comment by criss20ster** [comment]
ok I was looking wrong, I was looking in the game and this is what you need in the menu before starting the game

**Comment by XPiRiJenZ** [comment]
maybe also nice to move the text for field margin to the help-menu for further desription. the text there is also too long in the coursegenerator-menu

**Comment by Tensuko** [comment]
@XPiRiJenZ Sry that we won't do.
Reason: The mini Hud should only contain the information that is needed for visual adjusting width and offset to better see it and what is needed to start the driver.
Some quick access or the course delete action that was suggested is okay but when we start with more settings ppl will start to want more and that was not the reason why we created that menu.
Generator Settings stay in generator (workwidth is only there, as said, for a visuel feedback).

**Comment by XPiRiJenZ** [comment]
@Tensuko ich meinte das hier, weil es zu lang ist :) 
![InkedScreenshot 2022-01-26 21-41-05_LI](https://user-images.githubusercontent.com/43669988/151244303-ff686f00-7385-47c7-ae9a-25066c581753.jpg)

da hin
![InkedScreenshot 2022-01-26 21-41-16_LI](https://user-images.githubusercontent.com/43669988/151244326-e9c8a373-4e29-42c3-bcea-d32644dece15.jpg)

**Comment by rubesaca** [comment]
Same on Spanish translation.
![Spanish translation](https://user-images.githubusercontent.com/3020572/151247453-148ed590-59bd-4a27-885e-51d380cd234f.png)

**Comment by Tensuko** [comment]
@XPiRiJenZ oh, ja sry das habe ich dann falsch verstanden.
Ja das ist natürlich eine Option! Gute Idee.

Der Kursgenerator muss so oder so angepasst werden noch, vlt mach ich das gleich mit.

@rubesaca Find a shorter translation for it.
I have my doubts that there is no term to explain this in a shorter way...
offset elft/right ? in spanish ?
offset up/down ?

**Comment by criss20ster** [comment]
I found a short translation ,  how can i add xml file

**Comment by KCHARRO** [comment]
Ok, i find a way to solve this....

**Comment by KCHARRO** [comment]
Right now partially fixed long text issue #705.
It is complicated in such a small space, since in Spanish the words (for the most part) are longer than in other languages.
Example: 
en: Left/Right 
es: Izquierda/Derecha

When abbreviating them you have to be very careful, because then their meaning is not understood.

**Comment by Duseek** [comment]
Hello!
I change Russian translation - remove long text on miniHUD, translate convoy part and change some lowercase letters to uppercase. I hope this help you.
[This is link on Google Drive](https://drive.google.com/file/d/15sHbGmoyyb8rW-GynvcxPqCpWw-mvv-7/view?usp=sharing)

**Comment by Tensuko** [comment]
only with a pullrequest!

**Comment by Tensuko** [comment]
Translation related to every language. Need to find a way for this by the translater, sry.
